modules/update-duration-example.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/understanding-openshift-update-duration.adoc
+
+:_content-type: CONCEPT
+[id="update-duration-example_{context}"]
+= Example update duration of cluster Operators
+
+image::update-duration.png[A diagram displaying the periods during which cluster Operators update themselves during an OCP update]
+
+The previous diagram shows an example of the time that cluster Operators might take to update to their new versions.
+The example is based on a three-node AWS OVN cluster, which has a healthy compute `MachineConfigPool` and no workloads that take long to drain, updating from 4.13.0 to 4.14.0.
+
+[NOTE]
+====
+The specific update duration of a cluster and its Operators can vary based on several cluster characteristics, such as the target version, the amount of nodes, and the types of workloads scheduled to the nodes.
+====
+
+Each cluster Operator has characteristics that affect the time it takes to update itself.
+For instance, the Kube API Server Operator in this example took over eleven minutes to update because `kube-apiserver` provides graceful termination support, meaning that existing, in-flight requests are allowed to complete gracefully.
+This may result in a longer shutdown of the `kube-apiserver`.
+In the case of this Operator, update speed is sacrificed to help prevent and limit disruptions to cluster functionality during an update.
+
+Another characteristic that affects the update duration of an Operator is whether the Operator utilizes DaemonSets.
```

Comment:
This phrase applies to daemon sets for all Operators, yes?

```suggestion
Another characteristic that affects the update duration of an Operator is whether the Operator utilizes daemon sets, which can take time to roll out their version changes.
```

Reply:
@wking could confirm this. I'm personally not sure whether it applies to all Operators that utilize daemon sets, or only Operators that utilize "full-cluster daemon sets" (or whether that distinction makes sense, I really don't know anything about daemon sets haha)

Reply:
I agree with Seb.  There are some OpenShift DaemonSets that limit themselves to just control-plane nodes, like `openshift-machine-config-operator`'s `machine-config-server`.  The number of compute nodes does not affect the duration of updating that DaemonSet.  The machine-config operator also has a `machine-config-daemon` DaemonSet, which is full-cluster, and which updates with `maxUnavailable: 10%`.  But the time it takes to roll that DaemonSet is much less than the time it takes to roll the MachineConfigPools, so we don't usually bother calling out the `machine-config-daemon` roll time.

Reply:
Given that information, I think it might make sense to not swap the phrasing and mention only daemon sets in general here, while keeping "full-cluster DaemonSets, which can take time to roll out their version changes," down below (so to be more specific about how the full-cluster daemon sets in particular add a lot of time to an Operator's update duration).

@mburke5678 let me know if you still want me to swap the phrasing and make it work.